app/views/admin/settings/_update_setting_button.html.erb

```diff
@@ -1,3 +1,3 @@
 <% if current_user.has_role?(:super_admin) %>
-  <%= f.submit "Update Settings", class: "crayons-btn mt-4", data: { disable_with: false } %>
+  <%= f.submit "Update Settings", class: "crayons-btn mt-4", aria: { label: defined?(aria_label) ? aria_label : nil }, data: { disable_with: false } %>
```

Comment:
This allows a developer to specify a better button text than the generic `Update Settings`.